content-disposition/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for content-disposition 0.5
+// Project: https://github.com/jshttp/content-disposition
+// Definitions by: Stefan Reichel <https://github.com/bomret>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace contentDisposition {
+    export class ContentDisposition {
```

Comment:
The source code shows that this class is not exported, and is [commented](https://github.com/jshttp/content-disposition/blob/master/index.js#L439) as being `for v8 optimization`. So it would be better to just use the type `{ type: string, parameters: any }` inline.